app/services/candidate_interface/validate_course_query_string_params.rb

```diff
@@ -0,0 +1,46 @@
+module CandidateInterface
+  class ValidateCourseQueryStringParams
```

Comment:
This does more than validating course query params - perhaps rename it to reflect that (`ApplyFromFindPage`?)

Reply:
Yea good shout. There was a misunderstanding of the scope of what this would do. It used to validate query params in multiple controllers 👍